Configuration/PyReleaseValidation/python/relval_upgrade.py

```diff
@@ -89,6 +89,16 @@ def makeStepName(key,frag,step,suffix):
                 elif '2018' in key:
                     workflows[numWF+upgradeSteps['pixelTrackingOnly']['offset']] = [ upgradeDatasetFromFragment[frag], stepList['pixelTrackingOnly']]
 
+            # special workflows for HGCAL/TICL
+            if (upgradeDatasetFromFragment[frag]=="CloseByParticleGun"):
```

Comment:
can you restrict this to 2023 workflows? (We don't need to run TICL for Phase 1)

Reply:
Sure!